htdocs/class/xoopsform/formpassword.php

```diff
@@ -67,15 +76,17 @@ class XoopsFormPassword extends XoopsFormElement
      * @param string $value        Initial value of the field.
      *                             <strong>Warning:</strong> this is readable in cleartext in the page's source!
      * @param bool   $autoComplete To enable autoComplete or browser cache
+     * @param bool   $confirm      To enable the password confirmation
      */
-    public function __construct($caption, $name, $size, $maxlength, $value = '', $autoComplete = false)
+    public function __construct($caption, $name, $size, $maxlength, $value = '', $autoComplete = false, $confirm = false)
```

Comment:
Boolean flags like this are usually frowned on. See: https://www.martinfowler.com/bliki/FlagArgument.html

It is a sign we should have a new class.